### PR TITLE
Quicker convolution when groups=1

### PR DESCRIPTION
**Pull request by pfeatherstone**

### CODE REVIEW

**Comment by pfeatherstone** [comment]
@soumickmj Any thoughts on this?

**Comment by soumickmj** [comment]
Thanks @pfeatherstone for the contribution and sorry for the delay in merging.

File: torchcomplex/nn/functional.py [hunk 1]
```diff
@@ -63,20 +63,37 @@ def _fcaller(funtinal_handle, *args):
...
+            MrKr = funtinal_handle(inp_r, w_r, b_r, *args[3:]) #Real Feature Maps *(conv) Real Kernels
+            MiKi = funtinal_handle(inp_i, w_i, b_i, *args[3:]) #Imaginary Feature Maps * Imaginary Kernels
+            MrKi = funtinal_handle(inp_r, w_i, b_i, *args[3:]) #Real Feature Maps * Imaginary Kernels
+            MiKr = funtinal_handle(inp_i, w_r, b_r, *args[3:]) #Imaginary Feature Maps * Real Kernels
+
     real = MrKr - MiKi
     imag = MrKi + MiKr
     out = torch.view_as_complex(torch.stack((real,imag),dim=-1))
+
+    if not bias_added and b_r is not None and b_ri is not None:
```
Comment (pfeatherstone) [inline torchcomplex/nn/functional.py:93]: I've just noticed there is a mistake, it should be:
```b_i``` not ```b_ri```

Comment (pfeatherstone) [reply torchcomplex/nn/functional.py:93]: @soumickmj do you mind just committing to master rather than setting up another PR?

**Pull request merged**